sycl/doc/GetStartedGuide.md

```diff
@@ -278,6 +276,50 @@ command:
     c:\oclcpu_rt_<cpu_version>\install.bat c:\tbb_<tbb_version>\tbb\bin\intel64\vc14
     ```
 
+### Obtain prerequisites for ahead of time (AOT) compilation
+
+[Ahead of time compilation](CompilerAndRuntimeDesign.md#ahead-of-time-aot-compilation)
+requires OpenCL ahead of time compiler available in `PATH`. There is
+AOT compiler for each OpenCL device type (`GPU`, `CPU` and `ACC`
+(accelerator)).
+
+#### GPU
+
+* Linux
+
+  There are two ways how to obtain AOT GPU compiler `ocloc`:
```

Comment:
Can a user  on linux also use ocloc from oneAPI toolkit?

Reply:
No, OpenCL GPU runtime and ocloc is not a part of oneAPI toolkit on Linux. OpenCL GPU runtime, Level Zero and ocloc should be installed before oneAPI toolkit installation,